scripts/validate_billing_e2e_sessions.py

```diff
@@ -0,0 +1,35 @@
+import os
+import urllib.request
+import json
+import time
+import datetime
+
+def dateToMillis(date):
+    return int(time.mktime(date.timetuple()) * 1000)
+
+token = os.environ["MAPBOX_BILLING_STATISTICS_ACCESS_TOKEN"]
+
+periodEndTime = datetime.datetime.now()
+periodEndTimeMillis = dateToMillis(periodEndTime)
+
+periodStartTime = periodEndTime - datetime.timedelta(days=1)
+periodStartTimeMillis = dateToMillis(periodStartTime)
+
+print(f"Retrieving billing events for the time interval {periodStartTime} - {periodEndTime}")
+
+url = f"https://api.mapbox.com/billing/usage/v1?access_token={token}&period_start={periodStartTimeMillis}&period_end={periodEndTimeMillis}"
+
+response = urllib.request.urlopen(url)
+
+if response.status != 200:
+    raise ValueError(f"Request failed: {response.status}")
+
+data = json.loads(response.read().decode("utf-8"))
+
+numberOfSessions = data["nav2sestrip"]
+
+# We expect one session per requested period
+if numberOfSessions != 1:
```

Comment:
What do you think about an approach when tests write expected session count to a file, and then python script compares real usage with an expected result from the file, so that we don't need update test script when we update tests?

Reply:
Hmm, it sounds reasonable but raises some questions like:
- where to store this file
- how to update this file from Android Instrumentation tests environment
- set up authentication to read/update this file

It seems that this will complicate CI significantly anyway.

Given that we don't have plans of changing this value or tests logic, we can leave as is and change it if/when needed.